src/manifolds/MetricManifold.jl

```diff
@@ -544,9 +552,16 @@ arguments that will be passed to `OrdinaryDiffEq.solve`.
 Currently, the numerical integration is only accurate when using a single
 coordinate chart that covers the entire manifold. This excludes coordinates
 in an embedded space.
+
+!!! note
+    This function only works for Julia 1.1 or greater, when
+    [OrdinaryDiffEq.jl](https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl) is loaded with
+    ```julia
+    using OrdinaryDiffEq
+    ```
 """
 function solve_exp_ode(M, x, v, tspan; kwargs...)
-    error("solve_exp_ode not implemented on $(typeof(M)) for point $(typeof(x)), vector $(typeof(v)), and timespan $(typeof(tspan)). For a suitable default, enter `using OrdinaryDiffEq`.")
+    error("solve_exp_ode not implemented on $(typeof(M)) for point $(typeof(x)), vector $(typeof(v)), and timespan $(typeof(tspan)). For a suitable default, enter `using OrdinaryDiffEq` on Julia 1.1 or greater.")
```

Comment:
Could you add a small test for this error message? We do that for several errors with an `EmptyManifold` for manifold-vunctions as well.